app/models/case_request.rb

```diff
@@ -0,0 +1,39 @@
+# == Schema Information
+#
+# Table name: case_requests
+#
+#  id         :bigint(8)        not null, primary key
+#  meta       :jsonb
+#  created_at :datetime         not null
+#  updated_at :datetime         not null
+#  user_id    :bigint(8)        not null
+#
+# Indexes
+#
+#  index_case_requests_on_user_id  (user_id)
+#
+# Foreign Keys
+#
+#  fk_rails_...  (user_id => users.id)
+#
+
+class CaseRequest < ApplicationRecord
+  class InvalidPeriod < StandardError; end
+  belongs_to :user
+
+  def self.get_count_by_period(period = '7_DAY')
+    scope = case period
+    when '1_DAY'
+      CaseRequest.where(created_at: 1.day.ago..Time.current)
+    when '7_DAY'
+      CaseRequest.where(created_at: 7.days.ago..Time.current)
+    when '30_DAY'
+      CaseRequest.where(created_at: 30.days.ago..Time.current)
+    when 'ALL'
```

Comment:
instead of hardcoded period strings, how about the client just specifies the number of days where 0 == forever?

Reply:
Sure, that sounds good.

Just updated the the call so it now takes a `days_back` param `localhost:3000/api/case_requests?days_back=0` with `0` (or no `days_back` param being all time).

I also slightly updated the response to include the user's handle like so:
```
[{"user_id":2,"user_handle":null,"case_request_count":5},{"user_id":1,"user_handle":"sda","case_request_count":1}]
```
Just note that the user's handle isn't mandatory (unless you'd like it to be) so it might be `null` sometimes.